README.md

```diff
@@ -51,6 +52,8 @@ To test statusgo using a given network by number ID, use:
 make ci networkid=3
 ```
 
+If you have problems running tests on public network we suggest reading [e2e guide](https://github.com/status-im/status-go/blob/develop/e2e/README.md)
```

Comment:
Maybe we can use a relative link there? It won't be branch-dependent then...
Modern desktop tools for rendering Markdown support that as well.